register.py

```diff
@@ -1,14 +1,17 @@
 import pypandoc
-import os
 
 output = pypandoc.convert('README.md', 'rst')
-f = open('README.txt','w+')
+f = open('README.txt', 'w+')
 f.write(output)
 f.close()
 
 readme_rst = open('./README.txt').read()
-replace = '[SendGrid Logo]\n(https://uiux.s3.amazonaws.com/2016-logos/email-logo%402x.png)'
-replacement = '|SendGrid Logo|\n\n.. |SendGrid Logo| image:: https://uiux.s3.amazonaws.com/2016-logos/email-logo%402x.png\n   :target: https://www.sendgrid.com'
-final_text = readme_rst.replace(replace,replacement)
+replace = ('[SendGrid Logo]\n'
+           '(https://uiux.s3.amazonaws.com/2016-logos/email-logo%402x.png)')
+replacement = ('|SendGrid Logo|\n\n'
+               '.. |SendGrid Logo| image:: '
+               'https://uiux.s3.amazonaws.com/2016-logos/email-logo%402x.png\n'
+               '   :target: https://www.sendgrid.com')
```

Comment:
Too bad a multi-line string couldn't be used quite as well here since the url makes that one line 87 chars long (but no newline characters or extra brackets):

```python
replacement = """\
|SendGrid Logo|

.. |SendGrid Logo| image:: https://uiux.s3.amazonaws.com/2016-logos/email-logo%402x.png
   :target: https://www.sendgrid.com"""

```

Reply:
I went put a backslash in the multi-line comment to shorten the line. Maybe not the cleanest thing in the world, but I think it works better than the first attempt. Thoughts?

Reply:
@jlindenger ,

Ah of course! Nice one.